helm-charts/datacater/values.yaml

```diff
@@ -19,9 +19,9 @@ datacater:
   pythonrunner:
     name: "python-runner"
     image:
-      name: "datacater/python-runner"
+      repository: "datacater/python-runner"
       pullPolicy: IfNotPresent
-      version: "nightly-20230105"
+      version: "nightly"
```

Comment:
We use `nightly` as our `latest`-tag. Is the update of the version to `nightly` intentional despite using a specific nightly build for [other components](https://github.com/DataCater/datacater/blob/main/helm-charts/datacater/values.yaml#L14)?

Reply:
Will fix it. The reason was, that minikube could not find that tag. Slipped through :)
It is available, though https://hub.docker.com/r/datacater/python-runner/tags?page=1&name=nightly-20230105.